storage/volume/config.go

```diff
@@ -6,10 +6,10 @@ import (
 
 // Config represents volume config struct.
 type Config struct {
-	ControlAddress string
-	Timeout        time.Duration // operation timeout.
-	RemoveVolume   bool
```

Comment:
I am wondering if we could add some annotations for this field `RemoveVolume` ?

I wish that when code readers reader the struct itself, he could catch the exact meaning of it clearly, rather than trying hard to guess its meaning. 

If we make it, I think the lifetime of the code would be much longer to be alive.